src/travel-token-box/index.tsx

```diff
@@ -110,7 +110,8 @@ const TravelDeviceTitle = ({
     case 'mobile':
       return (
         <ThemeText type="heading__title" color="primary_2" style={styles.title}>
-          {inspectableToken.name}
+          {inspectableToken.name ||
+            t(TravelTokenTexts.toggleToken.unknownDevice)}
```

Comment:
Heller legg til denne teksten også i TravelTokenBoxTexts, selv om det blir duplikat. Vi har egentlig et 1-til-1 forhold mellom komponent og tekster, og det er kjekt å prøve å oppretteholde dette prinsippet så lenge det ikke blir for store mengder duplikate tekster.

Reply:
Den er god, fint å vite 👍 Oppdatert i 518276bd51811f3590732885f0d04385144f4b31